src/porepy/models/constitutive_laws.py

```diff
@@ -1658,12 +1664,15 @@ def advective_flux(
             Operator representing the advective flux.
 
         """
+        # The upwind discretization is non-linear (i.e. depends on solution state).
+        # Append to list of discretizations to be updated at each nonlinear iteration.
+        # self.add_nonlinear_discretization(discr)
+
```

Comment:
Should this be deleted? Intuitively, it seems the solution strategy is the better option.